src/main/controllers/AddUserRolesController.ts

```diff
@@ -30,12 +42,29 @@ export class AddUserRolesController extends RootController {
     }
 
     const roles = fields.roles;
-    await req.scope.cradle.api.registerUser({
-      email: fields._email,
-      firstName: fields._forename,
-      lastName: fields._surname,
-      roles: convertToArray(roles)
-    });
+    const serviceInfo = await this.serviceProviderService.getService(config.get('services.idam.clientID'));
+
+    if(fields._usertype === UserType.Support) {
+      await this.inviteService.inviteUser({
+        email: fields._email,
+        forename: fields._forename,
+        surname: fields._surname,
+        activationRoleNames: convertToArray(roles),
+        invitedBy: req.session.user.id,
+        clientId: serviceInfo.clientId,
+        successRedirect: serviceInfo.hmctsAccess.postActivationRedirectUrl
+      });
+    } else {
+      await this.inviteService.inviteUser({
+        email: fields._email,
+        forename: fields._forename,
+        surname: fields._surname,
+        activationRoleNames: convertToArray(roles),
+        invitedBy: req.session.user.id,
+        clientId: serviceInfo.clientId
```

Comment:
What would be the result of not passing any `successRedirect` here?

Reply:
The activate user stuff in api returns an AcrtivationResult object which has a redirect url in it which comes from this value. If the api returns null after activation then I think web public defaults to "https://www.gov.uk/" for the continue button